_clients/OpenSearch-dot-net.md

```diff
@@ -0,0 +1,395 @@
+---
+layout: default
+title: OpenSearch.Net client
+nav_order: 30
+has_children: false
+parent: .NET clients
+---
+
+# OpenSearch.Net client
+
+OpenSearch.Net is a low-level .NET client that provides the foundational layer of communicating with OpenSearch. It is dependency-free, and it can handle round-robin load balancing, transport, and the basic request/response cycle. OpenSearch.Net contains all OpenSearch API endpoints as methods. When using OpenSearch.Net, you need to construct the queries yourself.
```

Comment:
"of communication with"?

Reply:
I've sat here and read the sentence 10 times both ways.  Is it wrong the way Fanit has it written?  It sounds fine to me and I tried googling a bit but didn't see anything definitive either way (although I didn't look super long, to be fair).